.github/workflows/bump_submodules.yml

```diff
@@ -0,0 +1,46 @@
+name: Bump Submodule
+
+on:
+  schedule:
+    - cron: '0 0 * * 0,4'  # Run at 00:00 on Sunday and Thursday
+  pull_request:
+    branches: ["main"]
+    paths:
+      - ".github/workflows/bump_submodules.yml"
+
+jobs:
+  bump_submodule:
+    runs-on: ubuntu-latest
```

Comment:
nit: should match the rest of the project style on _not_ using ubuntu-latest but rather explicitly using `ubuntu-20.04`

The reasons there are mostly historical (https://opensource.google/documentation/reference/github/services#actions), but using the same style as the rest of the project will make updating easier.